src/main/java/controller/RateList.java

```diff
@@ -0,0 +1,98 @@
+package controller;
+
+import model.*;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/9/2019.
+ */
+public class RateList {
+
+    private static ArrayList<Rate> rateList = createRateList();
+    private static DecimalFormat df2 = new DecimalFormat("#.##");
+    private static Properties p = new Properties();
+
+    private static FileReader readProperties() throws FileNotFoundException {
+        return new FileReader(System.getProperty("user.dir") + "/src/main/resources/rateFilterData");
```

Comment:
it's better to read rate properties from classpath - more safe that be bounded to direct location of the file.

Reply:
done